src/main/java/com/marianbastiurea/lifeofbees/Hive.java

```diff
@@ -515,4 +460,28 @@ public void moveAnEggsFrameFromUnsplitHiveToASplitOne() {
     public void changeQueen() {
         queen = new Queen(0);
     }
-}
\ No newline at end of file
+
+    public void fillUpEggsFrame(Date currentDate, int numberOfEggs) {
+
+        int maxEggPerFrame = 6400; // a frame have around 8500 cells. 75% more or less are used by the queen to lay eggs.
```

Comment:
This is a property of the EggsFrame class, so move it to the EggsFrame class.

Reply:
done

Reply:
delete all maxEggPerFrame from all Hive class